samza-api/src/main/java/org/apache/samza/table/descriptors/RemoteTableDescriptor.java

```diff
@@ -76,6 +76,8 @@
   public static final String READ_RETRY_POLICY = "io.read.retry.policy";
   public static final String WRITE_RETRY_POLICY = "io.write.retry.policy";
   public static final String BATCH_PROVIDER = "io.batch.provider";
+  public static final String READ_RATE_LIMIT = "io.read.ratelimit";
```

Comment:
Minor: move them next to other rate limiter related configs (io.ratelimiter) etc.

Would it make sense to call these io.read.credits and io.write.credits for consistent terminology with rest of the configs?

Can you also document how this config is used?